sky/packages/sky/lib/src/services/fetch.dart

```diff
@@ -53,7 +54,7 @@ Future<UrlResponse> fetchUrl(String relativeUrl) async {
   UrlRequest request = new UrlRequest()
     ..url = url
     ..autoFollowRedirects = true;
-  return await fetch(request);
+  return fetch(request);
```

Comment:
fetch() and Future<UrlResponse> both return a Future<UrlResponse>, so this return just passes it through. The only reason you need an await is if you were doing some additional processing before returning. @johnmccutchan, could you please confirm or deny this?